src/bin/migrateDynamo.ts

```diff
@@ -38,49 +38,58 @@ program
 
 program
     .command('create [description]')
+    .option('--profile <string>', 'AWS credentials and configuration to be used', 'default')
```

Comment:
Why do we need a profile while creating a new migration script?

Reply:
When a migration script is created, it also creates the "MIGRATIONS_LOG" table in the Dynamo db against the AWS account which is denoted by the "profile" option.